tools/navcon-converter/navcon-converter

```diff
@@ -0,0 +1,139 @@
+#! /usr/bin/env python3
+#-*- coding: UTF-8 -*-
+
+# ===========================================================================
+#
+# Copyright (c) 2022 Unvanquished Developers
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+#
+# ===========================================================================
+
+import struct
+import sys
+
+"""
+Version 2 binary format
+
+Header version number then connection count then arrays of various types.
+
+Example:
+header count {x1 z1 y1 x2 z2 y2}[] rad[] flags[] areas[] dirs[] userids[]
+i      i      f  f  f  f  f  f     f     H       B       B      I
+
+int header;
+int count;
+union coords_t { float x1, z1, y1, x2, z2, y2 };
+coords_t[count] coords;
+float rad[count];
+unsigned short flags[count];
+unsigned byte areas[count];
+unsigned byte dirs[count];
+unsigned int userids[count];
+"""
+
+"""
+Version 3 text format
+
+Header line with prefix and integer version number
+then one connection per line, all numbers as integers.
+
+Example:
+#navcon 3
```

Comment:
I suspect our tokenizer may not accept `#navcon` as a token, so you might want to pick something else.

Reply:
Maybe just "navcon 3" would be enough

Reply:
done